internal/consensus/state.go

```diff
@@ -2348,6 +2352,11 @@ func (cs *State) addVote(
 		// Either duplicate, or error upon cs.Votes.AddByIndex()
 		return
 	}
+	if vote.Round == cs.Round {
```

Comment:
Would we be interested in also counting votes that came late? (i.e., from a previous round)
This could be useful to detect network "asynchrony" problems

Reply:
That's a good question. Without really blowing up the size of the metric tracking facilities, we wouldn't be able to store per-round information. We could, alternatively, store a simple 'number of late votes received since process start' value. I think that the benefit to that may be in determining how often the program is receiving late votes overall. Operators, seeing a spike in this value, would be able to determine that the chain was not operating synchronously. Thoughts?

Reply:
> 'number of late votes received since process start' 
Yeah, that seems a reasonable tradeoff to me... no need to have fine-grained info that blows up your storage. But some indication that something's going on if suddenly a lot of votes start coming late